services/ingestion/etl/inforegio/json/src/lib/transform.js

```diff
@@ -2,6 +2,12 @@
  * Transform message (REGIO JSON)
  */
 
+// Takes DD/MM/YYYY to MM/DD/YYYY
+const formatDate = date => {
+  const d = date.split(/\//);
+  return new Date(`${d[1]}/${d[0]}/${d[2]}`).toISOString();
```

Comment:
1. I'd check if `d.length === 3` before
2. Don't use strings with the `Date` constructor (see the notice: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date), it's safer to call `Date(d[2], d[1] - 1, d[0])`